core/src/main/java/org/mule/runtime/core/internal/el/ExpressionLanguageAdaptorHandler.java

```diff
@@ -70,7 +69,8 @@ public ExpressionLanguageAdaptorHandler(ExtendedExpressionLanguageAdaptor defaul
       expressionLanguages.put(MEL_PREFIX, mvelExpressionLanguage);
     }
 
-    exprPrefixPattern = compile(EXPR_PREFIX_PATTERN_TEMPLATE.replaceAll("LANGS", join(expressionLanguages.keySet(), '|')));
+    exprPrefixPattern =
+        Pattern.compile(EXPR_PREFIX_PATTERN_TEMPLATE.replaceAll("LANGS", join(expressionLanguages.keySet(), '|')));
```

Comment:
there is a general thing with your IDE against static imports it seems

Reply:
not this one. This is because not there's another symbol called compile in this class